<launch>
	<arg name="output" default="log"/>
	<arg name="publish_clock" default="true"/>
	<arg name="is_vulkan" default="true"/>
	<arg name="host" default="localhost" />
	<arg name="frame" default="/world_enu"/>
 	<arg name="use_octree"/>
	<node name="airsim_node" pkg="airsim_ros_pkgs" type="airsim_node" output="screen">
		<param name="is_vulkan" type="bool" value="false" /> 
		<!-- ROS timer rates. Note that timer callback will be processed at maximum possible rate, upperbounded by the following ROS params -->
		<param name="update_airsim_img_response_every_n_sec" type="double" value="0.05" /> 
		<param name="update_airsim_control_every_n_sec" type="double" value="0.01" />
		<param name="update_lidar_every_n_sec" type="double" value="0.01" />
		<param name="publish_clock" type="bool" value="$(arg publish_clock)" />
		<param name="world_frame_id" type="string" value="$(arg frame)" />
		<!-- world_enu -->
		<param name="odom_frame_id" type="string" value="odom_local_enu" />
		<param name="host_ip" type="string" value="$(arg host)" />
	</node>
	<node name="airsim_map_node" pkg="airsim_ros_pkgs" type="airsim_map_node" output="screen">
		<param name="host_ip" type="string" value="$(arg host)" />
		<param name="world_frame_id" type="string" value="$(arg frame)" />
		<param name="resolution" type="double" value="0.1" />
		<param name="use_octree" type="bool" value="$(arg use_octree)" />
	</node>

	<!-- Static transforms -->
	<include file="$(find airsim_ros_pkgs)/launch/static_transforms.launch"/>
</launch>
